rubix-spi/src/main/java/com/qubole/rubix/spi/CacheConfig.java

```diff
@@ -190,6 +198,16 @@ public static int getDiskReadBufferSize(Configuration conf)
     return conf.getInt(KEY_DISK_READ_BUFFER_SIZE, DEFAULT_DISK_READ_BUFFER_SIZE);
   }
 
+  public static int getHeartbeatInitialDelay(Configuration conf)
```

Comment:
Are there tests for Config ? Will you be adding tests ? Will code coverage fall by a few basis points ? ;)

Reply:
These methods are simple getters/setters for interacting with the Hadoop configuration, so I'm not sure whether unit testing each individual method would be useful here, other than to increase coverage numbers.
Should we adopt a strategy like Presto for verifying configurations? (Example : https://github.com/prestodb/presto/blob/master/presto-hive/src/test/java/com/facebook/presto/hive/TestHiveClientConfig.java)

Reply:
Yes. We should. However, if it doesnt already exist, then it is outside the scope of this PR. Can you please open an issue ?

Reply:
GH issue opened (#153) and JIRA issue added (QUB-74)